data/equipe.yml

```diff
@@ -45,4 +45,14 @@ membres:
     social:
       - title: github
         url: "https://gitlab.com/snakedye"
+
+  - name: "Yann Mijatovic"

```

Comment:
Hey the error commes from the indentation here. You can simply align it with other members.